tests/test_modeling_common.py

```diff
@@ -618,9 +618,8 @@ def check_same_values(layer_1, layer_2):
             config_tied.torchscript = False
             model_tied = model_class(config_tied)
             params_tied = list(model_tied.parameters())
-
-            # Check that the embedding layer and decoding layer are the same in size and in value
             self.assertGreater(len(params_not_tied), len(params_tied))
```

Comment:
should tying weights increase the number of parameters?

Reply:
I think if you look into the `tie_weights` function in `modeling_utils` you see why. The LM word embeddings are not tied but copied if one uses torchscript.

Reply:
Got it. The thing I am confused about is whether this assert is needed, since my code runs with `jit.trace`, as demonstrated by passing `test_torchscript*`.

Reply:
@LysandreJik any idea?

Reply:
I think this test is in place as a way to make sure the two layers are copied. This assert doesn't seem necessary, I think you can remove it.